CoreFoundation/Parsing.subproj/CFXMLInterface.c

```diff
@@ -172,29 +186,36 @@ _CFXMLInterfaceSAXHandler _CFXMLInterfaceCreateSAXHandler() {
 #if DEPLOYMENT_RUNTIME_SWIFT
     saxHandler->internalSubset = (internalSubsetSAXFunc)__CFSwiftXMLParserBridge.internalSubset;
     saxHandler->isStandalone = (isStandaloneSAXFunc)__CFSwiftXMLParserBridge.isStandalone;
-    
     saxHandler->hasInternalSubset = (hasInternalSubsetSAXFunc)__CFSwiftXMLParserBridge.hasInternalSubset;
     saxHandler->hasExternalSubset = (hasExternalSubsetSAXFunc)__CFSwiftXMLParserBridge.hasExternalSubset;
-    
+    // saxHandler->resolveEntity    replaced by xmlSetExternalEntityLoader
     saxHandler->getEntity = (getEntitySAXFunc)__CFSwiftXMLParserBridge.getEntity;
-    
+    saxHandler->entityDecl = (entityDeclSAXFunc)__CFSwiftXMLParserBridge.entityDecl;
     saxHandler->notationDecl = (notationDeclSAXFunc)__CFSwiftXMLParserBridge.notationDecl;
     saxHandler->attributeDecl = (attributeDeclSAXFunc)__CFSwiftXMLParserBridge.attributeDecl;
     saxHandler->elementDecl = (elementDeclSAXFunc)__CFSwiftXMLParserBridge.elementDecl;
     saxHandler->unparsedEntityDecl = (unparsedEntityDeclSAXFunc)__CFSwiftXMLParserBridge.unparsedEntityDecl;
+    // saxHandler->setDocumentLocator
     saxHandler->startDocument = (startDocumentSAXFunc)__CFSwiftXMLParserBridge.startDocument;
     saxHandler->endDocument = (endDocumentSAXFunc)__CFSwiftXMLParserBridge.endDocument;
-    saxHandler->startElementNs = (startElementNsSAX2Func)__CFSwiftXMLParserBridge.startElementNs;
-    saxHandler->endElementNs = (endElementNsSAX2Func)__CFSwiftXMLParserBridge.endElementNs;
+    // saxHandler->startElement     replaced by startElementNs
+    // saxHandler->endElement       replaced by endElementNs
+    // saxHandler->reference
     saxHandler->characters = (charactersSAXFunc)__CFSwiftXMLParserBridge.characters;
+    saxHandler->ignorableWhitespace = (ignorableWhitespaceSAXFunc)__CFSwiftXMLParserBridge.ignorableWhitespace;
     saxHandler->processingInstruction = (processingInstructionSAXFunc)__CFSwiftXMLParserBridge.processingInstruction;
+    saxHandler->comment = (commentSAXFunc)__CFSwiftXMLParserBridge.comment;
+    // saxHandler->warning
```

Comment:
Would you be able to clean up these commented lines or add more context around them please? When the final code is reviewed without any context of this PR, it won't be clear why they are commented out and what the implications of that are.

Reply:
Sure, will do so.

I sorted the fields as they are in _xmlSAXHandler and added all unused fields as comments to make it more obvious what is (not) used.
Do you think it is better to remove the unused fields or to add a short explanation above the block with “currently not used by CFXMLInterface” at these fields?

Reply:
@millenomi WDYT? What kind of clarification in code comments would you prefer?

Reply:
I just pushed a minor update that adds some comments.